src/index.html

```diff
@@ -19,9 +19,423 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
This link to the stylesheet 'styles/index.scss' seems to be incorrect. The .scss files are not directly linked in the HTML file. They are compiled into a single .css file which is then linked in the HTML. Please check your build process and make sure to link the correct .css file in your HTML.